pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -211,7 +211,7 @@ stdenv.mkDerivation ({
     configureFlags="${concatStringsSep " " defaultConfigureFlags} $configureFlags"
 
     # nativePkgs defined in stdenv/setup.hs
-    for p in "''${!nativePkgs[@]}"; do
+    for p in "''${nativePkgs[@]}"; do
```

Comment:
The keys of the map, instead of the values. (With arrays, really int maps, it gives you the indices.)